scout/server/blueprints/public/templates/public/index.html

```diff
@@ -75,10 +75,10 @@ <h1 class="display-4">Scout</h1>
       </div>
     </div>
     <p class="pb-1 pt-2">Collaborations:</p>
-    <div class="row bg-dark rounded-3">
+    <div class="row rounded-3">
       <div class="col-xs-12 mb-2 mt-2 col-sm-3">
         <a href="https://genomicmedicine.se/" target="_blank">
-          <img class="img-fluid mx-auto" width="250" alt="Genomic Medicine Sweden logotype" src="{{ url_for('public.static', filename='logo-gms.png') }}">
+          <img class="img-fluid mx-auto" width="250" alt="Genomic Medicine Sweden logotype" src="{{ url_for('public.static', filename='GMS-logo-dark.png') }}">
```

Comment:
It's not really dark, more transparent no?

Reply:
The technical term is positive, but I kept the syntax they had for some level of simplicity.